### PR TITLE
testik number 2

### PR DESCRIPTION
**Pull request by AghajanyanA**
test

### CODE REVIEW

File: Armen/daser andoni het/myMap.js [hunk 1]
```diff
@@ -30,4 +30,4 @@ function reversed(text) {
 
 reversed('1234321')
 
-// test
\ No newline at end of file
+// test aougphfb[]
```
Comment (Andranik199) [inline Armen/daser andoni het/myMap.js:33]: wtf ?

Comment (AghajanyanA) [reply Armen/daser andoni het/myMap.js:33]: fuc off

Comment (Eduard0998) [reply Armen/daser andoni het/myMap.js:33]: ya toje svami

**Comment by AghajanyanA** [comment]
test

**Pull request closed without merging**